data-node/src/main/java/org/graylog/datanode/OpensearchDistribution.java

```diff
@@ -28,4 +29,7 @@ public OpensearchDistribution(Path path, String version) {
         this(path, version, null, null);
     }
 
+    public Path getOpensearchBinFile() {
```

Comment:
Maybe name this `getOpenSearchBinDir` ? as with most other paths or really `File`?

Reply:
This is really the executable bin file. Maybe renaming to getOpensearchExecutable?

Reply:
Done, thx!